Dockerfile

```diff
@@ -42,13 +42,25 @@ RUN pipenv sync --system
 
 RUN chown -R spark_user:spark /opt/bitnami
 
-# Set up Jupyter directories
+# Set up Jupyter Lab directories
 ENV JUPYTER_CONFIG_DIR=/.jupyter
 ENV JUPYTER_RUNTIME_DIR=/.jupyter/runtime
 ENV JUPYTER_DATA_DIR=/.jupyter/data
 RUN mkdir -p ${JUPYTER_CONFIG_DIR} ${JUPYTER_RUNTIME_DIR} ${JUPYTER_DATA_DIR}
 RUN chown -R spark_user:spark /.jupyter
 
+# Set up Jupyter Hub directories
+ENV JUPYTERHUB_CONFIG_DIR=/srv/jupyterhub
+RUN mkdir -p ${JUPYTERHUB_CONFIG_DIR}
+COPY ./src/notebook_utils/startup.py ${JUPYTERHUB_CONFIG_DIR}/startup.py
+RUN chown -R spark_user:spark ${JUPYTERHUB_CONFIG_DIR}
+
+# Jupyter Hub user home directory
+RUN mkdir -p /home
```

Comment:
This is the standard home root dir for all users. I'd pick something else just in case

Reply:
👍

Reply:
:+1: